### PR TITLE
Add Café de Nata to shop=pastry

### PR DESCRIPTION
**Pull request by rskedgell**

### CODE REVIEW

File: data/brands/shop/pastry.json [hunk 1]
```diff
@@ -6,6 +6,19 @@
...
+    {
+      "displayName": "Café de Nata",
+      "locationSet": {"include": ["gb"]},
+      "tags": {
+        "brand": "Café de Nata",
+        "brand:wikidata": "Q112672158",
+        "cuisine": "portuguese",
+        "diet:vegan": "yes",
+        "name": "Café de Nata",
+        "shop": "pastry",
```
Comment (LaoshuBaby) [inline data/brands/shop/pastry.json:18]: 图片

Maybe bakery is more suitable for Café de Nata?

I search the name on taginfo and only found a pastry and a fast_food in London

**Review by LaoshuBaby — CHANGES_REQUESTED** [review]

File: data/brands/shop/pastry.json [hunk 2]
```diff
@@ -6,6 +6,19 @@
...
+        "website": "https://www.cafedenata.com/"
```
[the remark this answers is not in the document]
Comment (rskedgell) [reply data/brands/shop/pastry.json:19]: Noted, will remove. I haven't contributed to NSI for a while, it did last time I looked in any detail. Perhaps this should be added to the "Overview" page in a list of tags not to include?

File: data/brands/shop/pastry.json [hunk 3]
```diff
@@ -6,6 +6,19 @@
...
   },
   "items": [
+    {
+      "displayName": "Café de Nata",
+      "locationSet": {"include": ["gb"]},
+      "tags": {
+        "brand": "Café de Nata",
+        "brand:wikidata": "Q112672158",
+        "cuisine": "portuguese",
+        "diet:vegan": "yes",
```
[the remark this answers is not in the document]
Comment (rskedgell) [reply data/brands/shop/pastry.json:16]: Yes, it applies to all branches, which is why I added it.

File: data/brands/shop/pastry.json (hunk 1 above)
Comment (rskedgell) [reply data/brands/shop/pastry.json:18]: No objection, will change.

File: data/brands/shop/pastry.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (rskedgell) [reply data/brands/shop/pastry.json:19]: Noted, will remove.

[the remark this answers is not in the document]
Comment (LaoshuBaby) [reply data/brands/shop/pastry.json:19]: > Perhaps this should be added to the "Overview" page in a list of tags not to include?

see https://github.com/osmlab/name-suggestion-index/wiki/Contributing#tags-required

"Note for brands, that common tags with additional details such as 'website' or 'facebook' is not accepted directly."

**Pull request merged**